_security-plugin/access-control/document-level-security.md

```diff
@@ -124,3 +124,26 @@ PUT _plugins/_security/api/roles/abac
   }]
 }
 ```
+## Use term-level lookup queries (TLQs) with DLS 
+
+You can perform term-level lookup queries (TLQs) with document-level security (DLS) using either of two modes: adaptive or filter-level. The default mode is adaptive, where OpenSearch automatically switches between Lucene level or filter level mode depending on whether or not there is a term-level lookup query. DLS queries without TLQs are executed at the Lucene level mode, whereas DLS queries with TLQs are executed on filter level mode.
+
+By default, the security plugin detects if a DLS query contains a TLQ or not, and chooses the appropriate mode automatically at runtime.
+
+To learn more about OpenSearch queries, see [Term-level queries](https://opensearch.org/docs/latest/opensearch/query-dsl/term/).
+
+### How to set the DLS evaluation mode in `opensearch.yml`
+
+By default, the DLS evaluation mode is set to `adaptive`. You can also explicitly set the mode in `opensearch.yml` with the `plugins.security.dls.mode` setting. Add a line to `opensearch.yml` with the desired evaluation mode. 
+For example, to set it to filter level, add this line:
+```
+plugins.security.dls.mode: filter-level
+```
+
+#### DLS Evaluation modes
+
+| Evaluation mode | Parameter | Description | Usage |
+:--- | :--- | :--- | :--- |
+Lucene level DLS | `lucene-level` | This setting makes all DLS queries apply to the Lucene level. | Lucene level DLS modifies Lucene queries and data structures directly. This is the most efficient mode but does not allow certain advanced constructs in DLS queries, including TLQs.
+Filter level DLS | `filter-level` | This setting makes all DLS queries apply to the filter level. | In this mode, OpenSearch applies DLS by modifying queries that OpenSearch receives. This allows for term-level lookup queries in DLS queries, but you can only use the `get`, `search`, `mget`, and `msearch` operations to retrieve data from the protected index. Additionally, cross-cluster searches are limited with this mode.
+Adaptive | `adaptive-level` | The default setting that allows OpenSearch to automatically choose the mode. | DLS queries without TLQ are executed on Lucene level, while DLS queries that contain TLQ are executed on filter level.
```

Comment:
Please change the last sentence to "DLS queries without TLQs are executed in Lucene-level mode, while DLS queries that contain TLQs are executed in filter-level mode." This presumes that "TLQs" should be plural and that "in" is the appropriate preposition for "mode."

Reply:
great, thanks!